db/schema.rb

```diff
@@ -37,6 +37,7 @@
     t.bigint "order_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "quantity"
```

Comment:
I was thinking this may make things less complicated, at least for the cart- order item interaction.